src/components/item/ItemForm.vue

```diff
@@ -104,19 +171,24 @@ export default {
       }
     });
 
-    this.setEdittingItemIndex(null); 
+    this.setEdittingItemIndex(null);
+    this.setCurrentItemObj(null);
```

Comment:
why...

Reply:
mounted hook  is for doing something with DOM, also it could be some ticking because of reset after rendering.
Here you are doing something with data - created is hook for that.

Reply:
it's in unmounted...